src/components/templates/Jazzbar/JazzBar/JazzBar.module.scss

```
@@ -0,0 +1,13 @@
+@use "scss/attendee/variables.scss";
+
+.componentSpaceInfo {
+  text-align: center;
+  padding: 0 variables.$whitespace * 2;
+  margin: 0 auto;
+  width: 100%;
+  max-width: 420px;
+  h1 {
+    font-size: variables.$font-size-large;
+    font-weight: variables.$font-weight-normal;
```

Even if the old `constants.scss` is removed, it doesn't mean there weren't good parts and practices that can be reused.

```suggestion
    font-weight: variables.$font-weight--normal;
```